projects/helper/solana.js

```diff
@@ -1,135 +1,170 @@
-const axios = require('axios')
+const BigNumber = require("bignumber.js");
+const axios = require("axios");
 
-const endpoint = 'https://solana-api.projectserum.com/'
+const solscan_base = "https://public-api.solscan.io/account/";
```

Comment:
this was the only thing changed right?

Reply:
Nope, the solscan_base is used only to get balance of SOL, the endpoint is still used for all other queries (tokens, etc)